pysand/erosion.py

```diff
@@ -296,4 +296,54 @@ def probes(v_m, rho_m, Q_s, D, d_p, alpha=60, K=2e-9, n=2.6, rho_t=7850):
         C2 = 1
     C_unit = 3.15e10  # Conversion factor from m/s to mm/year (4.60)
     E = K * F(a_rad) * v_m ** n / (rho_t * At) * C2 * Q_s / 1000 * C_unit
+    return E
+
+
+def flexible(v_m, rho_m, mu_m, Q_s, mbr, D, d_p):
+    """
+    Particle erosion for flexible pipes with interlock carcass, model reference to DNVGL RP-O501, August 2015
+    :param v_m: Mix velocity [m/s]
+    :param rho_m: Mix density [kg/m3]
+    :param mu_m: Mix viscosity [kg/ms]
+    :param Q_s: Sand production rate [g/s]
+    :param mbr: Minimum Bending Radius in operation [# ID's]
+    :param D: Minimum internal diameter for the interlock carcass [m]
+    :param d_p: Particle diameter [mm]
+    :return: Erosion rate [mm/y]
+    """
+
```

Comment:
No validation for mbr?

Reply:
It has the same model restrictions, like max R = 50D.. So that's why I think we don't need to validate it locally? What do you think?

Reply:
I think we can leave it for now..

Reply:
Ok, agree